core/tests/performance_tests/base.py

```diff
@@ -45,28 +45,52 @@ def setUp(self):
         self.preload_option = None
 
     def _initialize_data_fetcher(self):
+        """Initializes SeleniumPerformanceDataFetcher instance."""
         self.data_fetcher = perf_services.SeleniumPerformanceDataFetcher(
             browser=perf_services.BROWSER_CHROME, username=self.username,
             preload_option=self.preload_option)
 
     def _get_complete_url(self, base_url, page_url_short):
+        """Returns the absolute URL by joining the base_url and page_url_short.
+
+        Args:
+            base_url: str. The base URL.
+            page_url_short: str. The relative page URL to be joined to base_url.
+
+        Returns:
+            str. The resulting joined URL.
+        """
         return urlparse.urljoin(base_url, page_url_short)
 
     def _load_page_to_cache_server_resources(self):
+        """Loads page for server side caching."""
         self.data_fetcher.load_url(self.page_url)
 
     def _record_page_metrics_from_uncached_session(self):
+        """Records the page metrics from uncached session for a given page
+        URL.
```

Comment:
Can we move `URL` to the above line as well?

Reply:
Actually it exceeds the limit in this case...
Thanks!